Config/CMakeIncludes/FindTasmanianDpcpp.cmake

```diff
@@ -15,10 +15,10 @@ if (NOT Tasmanian_dpcpp_compiler AND NOT Tasmanian_sycl_compiler)
     message(FATAL_ERROR "Tasmanian_ENABLE_DPCPP requires that the CMAKE_CXX_COMPILER is set to the Intel dpcpp compiler or a compatible sycl compiler.")
 endif()
 
-if (NOT DEFINED Tasmanian_MKL_SYCL_ROOT AND NOT "$ENV{Tasmanian_MKL_SYCL_ROOT}" STREQUAL "")
-    set(Tasmanian_MKL_SYCL_ROOT "$ENV{Tasmanian_MKL_SYCL_ROOT}" CACHE PATH "The root folder for the Intel oneMKL installation")
+if (NOT "${MKLROOT}" STREQUAL "")
```

Comment:
I think you can write
```cmake
if (NOT MKLROOT)

Reply:
> I think you can write
> 
> ```cmake
> if (NOT MKLROOT)
> ```

You're right. It doesn't work well with `$ENV{}` but it works when it is a regular variable.